x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/sub_feature_form.tsx

```diff
@@ -130,6 +131,15 @@ export const SubFeatureForm = (props: Props) => {
             props.onChange([...privilegesWithoutGroupEntries, selectedPrivilegeId]);
           }
         }}
+        legend={i18n.translate(
+          'xpack.security.management.editRole.subFeatureForm.controlLegendText',
+          {
+            defaultMessage: '{subFeatureName} privilege',
```

Comment:
This legend text would be a bit more accurate / less prone to cause confusion in the context of the rest of the page
```suggestion
            defaultMessage: '{subFeatureName} sub-feature privilege',
```

Reply:
Updated in 4401f4d

Reply:
Responded in https://github.com/elastic/kibana/pull/81499#discussion_r513664918.